ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -22,10 +23,11 @@ networking:
   networkType: ${NETWORK_TYPE}
 platform:
   openstack:
-    cloud:            ${OS_CLOUD}
-    externalNetwork:  ${OPENSTACK_EXTERNAL_NETWORK}
-    computeFlavor:    ${OPENSTACK_COMPUTE_FLAVOR}
-    lbFloatingIP:     ${LB_FIP_IP}
+    cloud:             ${OS_CLOUD}
+    externalNetwork:   ${OPENSTACK_EXTERNAL_NETWORK}
+    computeFlavor:     ${OPENSTACK_COMPUTE_FLAVOR}
+    lbFloatingIP:      ${LB_FIP_IP}
+    ingressFloatingIP: ${INGRESS_FIP_IP}
```

Comment:
We introduced `ingressFloatingIP` in 4.6 with https://github.com/openshift/installer/pull/3855

If there is a chance the step registry runs an installer < 4.6, we should use the same approach as [in shiftstack-ci
](https://github.com/shiftstack/shiftstack-ci/blob/main/run_ocp.sh) to be backward compatible and setup the ingress VIP after the deployment.

Reply:
Alternatively, we detect the installer version and test both code path.

Reply:
Good catch.

Reply:
Added a big fat note in the ref description; let's tackle that problem later. We can always keep using the template-based job until we figure out.